sakuya/wordle.py

```diff
@@ -122,6 +123,10 @@ async def guess(self, ctx: commands.Context, guess: str):
             await ctx.send("Your guess must be 5 letters, a-z only.")
             return
         guess = guess.lower()
+        if state.game_start != current_game_start() and state.in_progress:
```

Comment:
Instead of additional state, you can check if between 1-5 guesses have been made & if the last guess is different from the solution. If that's the case, then the last game must have been unfinished! I would suggest moving this into the if-statement below as well, right before the debug code.

Reply:
I just realised @mcpower already commented a better way to do this than what I suggested. A `finished()` method is a great idea!